sycl/test-e2e/ESIMD/unified_memory_api/gather_acc_dg2_pvc.cpp

```diff
@@ -0,0 +1,43 @@
+//==------- gather_acc_dg2_pvc.cpp - DPC++ ESIMD on-device test ------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: gpu-intel-dg2 || gpu-intel-pvc
+
+// Use per-kernel compilation to have more information about failing cases.
+// RUN: %{build} -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %{run} %t.out
+
+// The test verifies esimd::gather() functions accepting ACCESSOR
+// and optional compile-time esimd::properties.
+// The gather() calls in this test do not use cache-hint properties
+// or VS > 1 (number of loads per offset) to not impose using PVC features.
```

Comment:
Commit is not accurate I think in the part about not using PVC features

Reply:
Yes, fixed. Thank you.